app/src/main/java/net/skyscanner/backpack/demo/meta/ViewStory.kt

```diff
@@ -30,5 +29,4 @@ import javax.annotation.concurrent.Immutable
 annotation class ViewStory(
   val name: String = "Default",
   val screenshot: Boolean = true,
-  @LayoutRes val layoutId: Int = 0,
```

Comment:
If you try to use it with Android Studio, you'll get the following warning: "Resource IDs will be non-final by default in Android Gradle Plugin version 8.0, avoid using them as annotation attributes". Therefore I'm removing this attribute (and this also simplifies the Visitors)

Reply:
how are view stories defined then? the sample specifies it's a view story, but it's actually a compose story.
also with this gone, maybe we just have a `Story` annotation which has a type for view vs compose?

Reply:
Having separate annotations seems to be an easier way to use rather than specifying the type of each type.

And all the stories are defined as Composables, it's just you use AndoidView interop for view.